### ISSUE TITLE
Configuration of the 1-wire drivers won't be needed

### ISSUE THREAD

**Issue by starksm64** [burrsutter/rpi_DS18B20#2]
The raspbian image I will burn onto the sdhc card we give out will already have the 1-wire overlay and drivers enabled, so the user will not have to actually make that change and reboot their device to get going. We can mention such a change is necessary over the default raspbian distribution, but they won't have to do it.

**Comment by starksm64** [comment]
Change is in pull request #3.

**Comment by burrsutter** [comment]
Makes sense to me - I was mostly just taking notes so I could remember what I had done :-)
